Chafu/PhotoGalleryDataSource.cs

```diff
@@ -46,11 +47,12 @@ public class PhotoGalleryDataSource : BaseAlbumDataSource, IPHPhotoLibraryChange
         /// </summary>
         /// <param name="albumView"><see cref="AlbumView"/> attached to</param>
         /// <param name="cellSize"><see cref="CGSize"/> with cell size. If <see cref="CGSize.Empty"/> it will default to 100x100</param>
-        public PhotoGalleryDataSource(AlbumView albumView, CGSize cellSize)
+		public PhotoGalleryDataSource(AlbumView albumView, CGSize cellSize, MediaType mediaTypes)
```

Comment:
Please add default value to the ctor and add to the comment above.

Convert tab to 4 spaces